package.json

```diff
@@ -8,8 +8,19 @@
     },
     "main": "dist/index.js",
     "types": "dist/index.d.ts",
+    "module": "esm/index.js",
+    "exports": {
+        ".": {
+            "import": "./esm/index.js",
+            "require": "./index.js"
+        },
+        "./mui": {
+            "import": "./esm/mui.js",
+            "require": "./mui.js"
+        }
```

Comment:
Ok but we haven't the types that automatically resolve to `compat.d.ts` and `mui_compat.d.ts` with older typescript version...

Reply:
Does this make sence to you?  

<img width="381" alt="image" src="https://user-images.githubusercontent.com/6702424/188321921-6bdbd43b-8ad5-4a01-9002-1ebc0b6cc9b1.png">


I mean do you think i'm on the right path with this?

Reply:
@garronej it makes sense if you want to use the compact types when typescript in consuming project is less than 4.4, but keep in mind that as you mentioned earlier this is only compile time, only the types changes, the implementation won't be the compact ones.